### ISSUE TITLE
Workshop archive needs accessibility improvements

### ISSUE THREAD

**Issue by TimothyGu** [uclaacm/hack.uclaacm.com#229]
* [x] Quarter headings and event headings are both `h2`s. This is confusing, since events are subordinate to quarters. Event headings should be `h3`, while individual sessions of the event should be `h4`.

* [x] It's not clear to accessibility technologies (AT) that tags should be treated as tags. Even though they are _visually_ displayed as pills, AT don't know that, and tags appear as merely a heap of words.
   * [x] We need to add `role="list"` to containers of a collection of tags, and `role="listitem"` to each tag.
   * [x] We need to add `aria-label="Tags"` to containers of a collection of tags.

* [x] The "launch" button next to each event heading is an HTML `label` element. The `label` element should only be used in forms, not for a link. 

   ![image](https://user-images.githubusercontent.com/1538624/120099656-b37bc980-c10a-11eb-97f3-ca8eb5a2794b.png)

- [x] The accordion buttons have a weird label. We should make sure neither "Launch" nor "javascript" is included in the label for that button.

   ![image](https://user-images.githubusercontent.com/1538624/120099729-25541300-c10b-11eb-848a-4ce412204c0e.png)

* [x] It's unclear to AT what the `div` for each event is. As far as it sees, the `div` is just a generic container. We need to add `aria-label` or `aria-labelled-by` with the title of the event.

   ![image](https://user-images.githubusercontent.com/1538624/120099714-0c4b6200-c10b-11eb-9ea0-7c7a0b3ed6e7.png)
